package.json

```diff
@@ -277,16 +280,17 @@
     "analyze-bundle": "gulp js --production && source-map-explorer ./dist/application.js"
   },
   "devDependencies": {
-    "@babel/cli": "^7.2.3",
-    "@babel/core": "^7.3.4",
+    "@babel/cli": "^7.0.0",
```

Comment:
So these changes to babel versions shouldn’t be here—and this isn’t a yarn issue, it’s just a git issue. I think you might have inadvertently reverted some of master’s changes to `package.json` during a rebase?